### PR TITLE
Added Daily Wire presence

### PR DESCRIPTION
**Pull request by N0chteil**
Proof:
Screenshot_397
Screenshot_398
Screenshot_399
Screenshot_400

https://github.com/PreMiD/Presences/issues/3743

### CODE REVIEW

**Review by ririxidev — CHANGES_REQUESTED** [review]
support for watching episodes etc?

**Comment by N0chteil** [comment]
> support for watching episodes etc?

I have no subscription

**Review by ririxidev — APPROVED** [review]
Well, fine.

File: websites/D/Daily Wire/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  },
+  "url": "www.dailywire.com",
+  "version": "1.0.6",
+  "logo": "https://i.imgur.com/BevxEyq.png",
+  "thumbnail": "https://i.imgur.com/4eXamxn.png",
+  "color": "#000",
+  "category": "other",
+  "tags": [
+    "cars"
+  ],
```
Comment (Bas950) [inline websites/D/Daily Wire/dist/metadata.json:19]: Add some more tags not just "cars"

File: websites/D/Daily Wire/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  "author": {
+    "name": "N0chteil ^^",
+    "id": "495901098926669825"
+  },
+  "service": "Daily Wire",
+  "description": {
+    "en": "The Daily Wire is an American conservative news website and media company founded in 2015 by political commentator Ben Shapiro and director Jeremy Boreing. The site has been criticised for promoting misinformation."
+  },
+  "url": "www.dailywire.com",
+  "version": "1.0.6",
```
Comment (Bas950) [inline websites/D/Daily Wire/dist/metadata.json:12]:
```suggestion
  "version": "1.0.0",
```

File: websites/D/Daily Wire/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "author": {
+    "name": "N0chteil ^^",
+    "id": "495901098926669825"
+  },
+  "service": "Daily Wire",
+  "description": {
+    "en": "The Daily Wire is an American conservative news website and media company founded in 2015 by political commentator Ben Shapiro and director Jeremy Boreing. The site has been criticised for promoting misinformation."
+  },
```
Comment (Bas950) [inline websites/D/Daily Wire/dist/metadata.json:10]:
```suggestion
  "description": {
    "en": "The Daily Wire is an American conservative news website and media company founded in 2015 by political commentator Ben Shapiro and director Jeremy Boreing. The site has been criticised for promoting misinformation.",
    "nl": "The Daily Wire is een Amerikaanse conservatieve nieuwswebsite en mediabedrijf opgericht in 2015 door politiek commentator Ben Shapiro en regisseur Jeremy Boreing. De site is gekritiseerd vanwege het promoten van valse informatie."
  },
```

File: websites/D/Daily Wire/presence.ts [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    }
+  } else if(urlpath[1] === 'show') {
+    presenceData.details = "Viewing Show";
+    presenceData.state = document.querySelector("h3.css-1qqpwph")?.textContent || "Unknown";
+
+    if (setting.showButtons) {
+      presenceData.buttons = [
+        {
+          label: "View Show",
+          url: window.location.href
```
Comment (Bas950) [inline websites/D/Daily Wire/presence.ts:52]:
```suggestion
          label: "Watch Show",
          url: window.location.href
```

File: websites/D/Daily Wire/presence.ts [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    }
+  } else if(urlpath[1] === 'episode') {
+    presenceData.details = document.querySelector("h2.css-n0lwas.e172hw750")?.textContent || "Unknown";
+    presenceData.state = document.querySelectorAll("span.css-e6rih1.ew91t7w0")[1]?.textContent || "Unknown";
+
+    if (setting.showButtons) {
+      presenceData.buttons = [
+        {
+          label: "View Episode",
+          url: window.location.href
```
Comment (Bas950) [inline websites/D/Daily Wire/presence.ts:40]:
```suggestion
          label: "Watch Episode",
          url: window.location.href
```

File: websites/D/Daily Wire/presence.ts [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+    presenceData.details = "Home";
+  else if(urlpath[1] === 'news' && urlpath[2]) {
+    presenceData.details = "Viewing Article";
+    presenceData.state = document.querySelector("h1.css-cmdiie.e172hw750")?.textContent || "Unknown";
+
+    if (setting.showButtons) {
+      presenceData.buttons = [
+        {
+          label: "View Article",
+          url: window.location.href
```
Comment (Bas950) [inline websites/D/Daily Wire/presence.ts:28]:
```suggestion
          label: "Read Article",
          url: window.location.href
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**